clients/bolt-sdk/src/world/transactions.ts

```diff
@@ -48,7 +47,6 @@ export async function InitializeNewWorld({
   return {
     transaction: new Transaction().add(initializeWorldIx),
     worldPda,
-    worldId,
```

Comment:
on the high level API there is no function that takes worldId as parameter, so I'm removing it for now because it could cause confusion?